### ISSUE TITLE
The character flies in the air after a few seconds.

### ISSUE THREAD

**Issue by thegreatsai** [endink/Mediapipe4u-plugin#39]
### MediaPipe4U Version

20230316

### UE Version

5.0.x

### UnrealEngine Type

Compiled From Source Code

### What happened?

When I import a custom character in the demo project and follow the exact setup you did with the mannequin, the custom character works just fine. I tried to do it in a different project file and everything worked but within a few seconds of turning on the camera, the character seems to start flying in the air. I went back and forth and recreated all the steps but nothing seemed to solve the problem.

I have added a link to a folder containing two videos showing what I am talking about.

[https://drive.google.com/drive/folders/1a14f-623EsApE0MHTzH2pnPDQ1G5W-U5?usp=share_link](url)

**Comment by endink** [comment]
I checked your video. I didn't see your webcam graph, so here's what I found from the video I saw：

1. You must understand the "location calibration", which may seem a little confusing because the documentation is Google translate, sorry for that: [Location Solver Animation Node](https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/features/location_solver.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp)， **CalibrateCountdownSeconds** property is important.

2. If you don't want to move your character,  you can set the **SolveLocation** property of the MediaPipeAnimationInstance (your animation blueprint) to **false**.
document is here: [Control Your Solver](https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/advance/solver_bp.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp), please read the section "**switch solver**".   
   
   

> :blue_book:Simply, although the documentation is not friendly to English, you should still read it and understand the usage against the demo project, because of the complexity of the plugin, this is not a plugin that can be used without reading the document.

**Comment by thegreatsai** [comment]
SolveLocation solved the problem. The character didn't fly away. I have another question. Version 5.0 has an exe file in the Binary folder which launch the plugin with the character of my choosing. The problem is by default it selects Device Id 0 but I use a camera with DeviceID 3. I tried to change in BP_Mediapiperuntime but nothing works in the exe. In the editor, it detects the change. Is there any way I could include the option to change the device id after running the exe and from the window just like other options like record bvh? I played with the BP to integrate such a thing but no luck. A little bit of direction will be helpful :)
